server/db/product_models/gender.js

```diff
@@ -0,0 +1,29 @@
+const Sequelize = require('sequelize');
+const db = require('../database');
+
+const { ENUM, INTEGER } = Sequelize;
+
+const Genders = db.define(
```

Comment:
probably could include a gender enum in the product table and remove this table

Reply:
JL